pages/wishlist_page/wishlist.css

```diff
@@ -0,0 +1,35 @@
+.wishlist-main{
+    display: flex;
+    gap: 1rem;
+    flex-flow: row wrap;
+    justify-content: space-evenly;
+}
+.card-container{
+    max-width: 18%;
+    height: auto;
+    background-color: var(--st-white);
+    padding: 1rem;
+    margin: auto;
+}
+.prod-img{
+    max-width: 100%;
+    position: relative;
+}
+.wishlist-btn{
+    position: absolute;
+    top: 0.5rem;
+    right: 0.5rem;
+    color: var(--st-mid-red);
+    background-color: var(--st-white);
+    padding: 5px 0.5rem;
```

Comment:
Dont use different units for same property.

Reply:
okay

Reply:
done